### PR TITLE
Db/merge no docker

### PR DESCRIPTION
**Pull request by derekwbrown**
### What does this PR do?

Allows code to be built without the 'docer' tag

### CODE REVIEW

File: pkg/util/container/providers_stub.go [hunk 1]
```diff
@@ -0,0 +1,26 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2017 Datadog, Inc.
```
Comment (hkaj) [inline pkg/util/container/providers_stub.go]: nit but it's 2018 🎉

**Review by hkaj — DISMISSED** [review]

File: pkg/collector/autodiscovery/autoconfig.go [hunk 2]
```diff
@@ -466,7 +466,7 @@ func (ac *AutoConfig) GetChecks(config check.Config) ([]check.Check, error) {
 		if len(res) != 0 {
 			return res, nil
 		}
-		log.Debugf("%v: unable to load the check '%s': %s", loader, config.Name, err)
+		log.Errorf("%v: unable to load the check '%s': %s", loader, config.Name, err) // TODO: temp - delete me
```
Comment (hkaj) [inline pkg/collector/autodiscovery/autoconfig.go]: This was temporary, it should go back to a `Debugf`.

**Review by hkaj — CHANGES_REQUESTED** [review]

File: pkg/util/container/providers_stub.go (hunk 1 above)
Comment (derekwbrown) [reply pkg/util/container/providers_stub.go]: Heh.  Copy/paste error.

**Review by hkaj — APPROVED** [review]
Lgtm

**Pull request merged**